packages/storybook/stories/user-profile.mdx

```diff
@@ -0,0 +1,35 @@
+import { UserProfile } from '@equisoft/design-elements-react';
+import { ArgTypes, Meta, Canvas } from '@storybook/blocks';
+import * as UserProfileStories  from './user-profile.stories';
+
+<Meta of={UserProfileStories} />
+
+# User Profile
+1. [Definition](#definition)
+2. [Usage](#usage)
+3. [Variants](#variants)
+4. [Properties](#properties)
+
+## Definition
+User profile is a high-level navigation menu regrouping user settings and personal details [1]. 
+<Canvas of={UserProfileStories.Desktop} />
+
+## Usage
+### When to use
+- Use to display links to settings and personal details like user profile (name, etc.), login and security (username, password, pin), language and currency settings, appearance (light/dark theme, accessibility, font size), other settings and log out [1]. 
```

Comment:
Est-ce qu'il y aurait moyen de rendre ca plus concis un peu ou de revoir la formulation?
"_Used to showcase personal details such as the user's name and provide links to personal settings such as security, language, currency, and appearance preferences, as well as the option to log out._"